tests/scripts/remote_monitoring_tests/rest_apis/test_update_recommendations.py

```diff
@@ -5,6 +5,132 @@
 from helpers.utils import *
 
 
+@pytest.mark.sanity
```

Comment:
@msvinaykumar - You can use the term tests and other existing tests itself to validate the plots, update the json schemas and utils function to validate recommendations

Reply:
It appears that the TermTest is utilizing the validate_container function, which means that whatever TermTest validates, plots as well

Reply:
@khansaad - json schemas need updation to validate plots

Reply:
Updated all the schemas